trinity/protocol/les/exchanges.py

```diff
@@ -11,20 +11,27 @@
 from trinity.protocol.common.exchanges import (
     BaseExchange,
 )
+# Q: What is the order to imports?  and why sometimes new line between imports and sometimes not?
```

Comment:
ordering is:

1. standard library
2. 3rd party
3. local library
4. local modules (these are all `from . import` style imports

Within each section we alphabetize and *normally* do whitespace breaks between libraries.

Within each of those sections

Reply:
ah, great!  I wonder if my ide can set this order.  I've done it in the past for my IntelliJ.